pages/linux/setenforce.md

```diff
@@ -0,0 +1,14 @@
+# setenforce
+
+> Toggle SELinux between enforcing and permissive modes.
+> To enable or disable SELinux, edit `/etc/selinux/config` instead.
+> See also: `getenforce`, `semanage-permissive`.
+> More information: <https://manned.org/man/setenforce>.
```

Comment:
Are 4 lines of description supported?

Reply:
I recall people having a similar discussion to this back in #9507. But taking a quick look I'm not so sure what the final consensus was. I'll let others chip in with their opinions.

Reply:
> Are 4 lines of description supported?

Yep, while we do prefer having 3 lines of description we do allow four or more line in special cases like pacman, etc (where additional context improves the page's usefulness).